src/assets/i18n/en.json

```diff
@@ -0,0 +1,472 @@
+      "email": "Email",
+      "languages": "Languages"
+    }
+  },
+  "navigation": {
+    "menu": {
+      "title": "MENU",
+      "offers": "Offers",
+      "my_portfolio": "My portfolio",
+      "wallet": "Wallet",
+      "investment_groups": "Investment groups"
+    },
+    "settings": {
+      "title": "SETTINGS",
+      "user_profile": "User profile",
+      "payments": "Payments"
+    },
+    "admin": {
+      "title": "ADMIN",
+      "token_issuer.title": "TOKEN ISSUER",
+      "platform_manager.title": "PLATFORM MANAGER",
+      "withdrawals": "Withdrawals",
+      "deposits": "Deposits",
+      "activations": "Activations",
+      "bank_account": "Bank account",
+      "platform_roles": "Platform roles",
+      "platform_config": "Platform config"
+    },
+    "info": {
+      "contact_us": "Contact us",
+      "logout": "Log out"
+    },
+    "tooltip": {
+      "not_verified": "Profile not verified. Go to User Profile screen to finish onboarding process.",
+      "wallet_not_initialized": "In order to manage funds, you need to have your wallet initialized.",
+      "verification_pending": "Please wait for the administrator to verified your wallet."
+    }
+  },
+  "user_state_reminder": {
+    "not_verified": "Profile not verified. Please go | here | to verify your profile and finish onboarding procedure.",
+    "initialize_wallet": "Thank you for verifying your profile. | In order to manage funds, you need to have your wallet initialized. Please initialize your wallet | here | .",
+    "activation_pending": "Thank you for initializing your wallet.",
+    "activation_pending.user_notice": "Please wait for the administrator to get it verified.",
+    "activation_pending.admin_notice": "Please wait for a few moments to get everything set up."
+  },
+  "errors": {
+    "registration": {
+      "sign_up_invalid": "Invalid sign up data.",
+      "user_exists": "The given email is already in use.",
+      "confirmation_token_invalid": "Invalid email confirmation data.",
+      "confirmation_token_expired": "Confirmation link has expired.",
+      "social_failed": "There is a problem with social login method. Try again later.",
+      "captcha_failed": "You have failed captcha validation. Please, try again."
+    },
+    "auth": {
+      "invalid_login_method": "Invalid login method",
+      "invalid_forgot_password_link": "Invalid forgot password link. Request a new one.",
+      "invalid_credentials": "Invalid username or password."
+    },
+    "user": {
+      "no_user": "The requested user does not exist.",
+      "invalid_bank_account_data": "Bank account data is invalid.",
+      "different_password": "Old password is invalid.",
+      "invalid_privilege": "You have no sufficient privilege to do this action."
+    },
+    "wallet": {
+      "missing_wallet": "Your wallet is missing.",
+      "cannot_create_new_wallet": "A new wallet cannot be created.",
+      "not_enough_funds": "You have not enough funds on your wallet.",
```

Comment:
Needs to be rephrased

Reply:
Good points, but I won't make changes here in this pull request. I just copied values from the template. There is room for improvements in phrases.

Reply:
Nah, you're right. If you have some time, you can point out more grammar issues in English and I'll make changes here. This will make us look more professional 😎 .

Reply:
Lol, sure thing 😄 
I'm gonna drop a comment if I notice more issues